site/plugins/site/extensions/fieldMethods.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Kirby\Toc\Toc;
+
+return [
+    'toToc' => function ($field, string $headline = 'h2') {
+        return Toc::headlines($field, $headline);
+    },
+    'withAnchorHeadlines' => function ($field, string $headlines = 'h2|h3') {
+        return Toc::anchors($field, $headlines);
+    },
+    'withCode' => function ($field) {
+        $field->value = preg_replace(
+            '$\`(.*?)\`$',
+            '<code>$1</code>',
+            $field->value
+        );
+        return $field;
+    },
+
+    /* Legacy */
+    'replace' => function ($field, $replace) {
+        $field->value = Str::template($field->value(), $replace);
```

Comment:
We can use all the classes we use in the plugin with the full class name instead of alias?

Reply:
I am not sure what you mean here 😬

Reply:
😅 I mean it's not a bug, it's a standard. Let's import the classes we use alias.

Reply:
Now understood! :)

Reply:
@distantnative There are uses that need to be imported in other files.